drift/core/psestimation.py

```diff
@@ -592,31 +597,28 @@ def fisher_bias(self):
 
     # ====== Estimate the q-parameters from data ========
 
-    def q_estimator(self, mi, vec1, vec2=None, noise=False):
-        """Estimate the q-parameters from given data (see paper).
-
+    def project_vector_kl_to_sky(self, mi, vec1, vec2=None):
+        """
         Parameters
         ----------
         mi : integer
-            The m-mode we are calculating for.
-        vec : np.ndarray[num_kl, num_realisatons]
-            The vector(s) of data we are estimating from. These are KL-mode
-            coefficients.
-        noise : boolean, optional
-            Whether we should project against the noise matrix. Used for
-            estimating the bias by Monte-Carlo. Default is False.
+            M-mode index to fetch for.
+        vec : np.ndarrays
```

Comment:
Need to update for `vec1`, `vec2` parameters

Reply:
done.